env0/resource_project.go

```diff
@@ -110,7 +109,18 @@ func resourceProjectUpdate(ctx context.Context, d *schema.ResourceData, meta int
 	apiClient := meta.(client.ApiClientInterface)
 
 	id := d.Id()
-	var payload client.ProjectCreatePayload
+	var payload client.ProjectUpdatePayload
+
+	if d.HasChange("parent_project_id") {
+		parentProjectId := d.Get("parent_project_id").(string)
+		if parentProjectId == "" {
```

Comment:
This was not supported in the UI.
If it's valid to make a subproject a project (with no parent), let me know, and I will modify this.

Reply:
lmk if that makes sense to you or if there are any further questions 👍

Reply:
makes sense. I'll fix that.

Reply:
@ItamarMalka - I fixed it: updated code, tests, and integration tests.
Now support moving a subproject to become a project.